### ISSUE TITLE
V0.1.27 fails on Termux on Android

### ISSUE THREAD

**Issue by MrC0D3** [yamadashy/repopack#56]
```
$ repopack

📦 Repopack v0.1.27

✖ Error during packing
Unexpected error: Expected `concurrency` to be an integer from 1 and up or `Infinity`, got `0` (number)
For more help, please visit: https://github.com/yamadashy/repopack/issues
```

Reviewing releases, concurrency was introduced a few versions ago in v0.1.24. Installing v0.1.23 results in a successful repopack.

```
$ repopack

📦 Repopack v0.1.23

✔ Packing completed successfully!
```

**Comment by yamadashy** [comment]
Thanks for reporting this issue. I'll investigate the CPU detection problem and fix it in the next release. Your feedback is really helpful!

**Comment by yamadashy** [comment]
This issue has been fixed in v0.1.28.

The new version sets the default concurrency to 1 when the CPU count is unavailable, which should resolve the problem on Termux and similar environments.

Release notes: https://github.com/yamadashy/repopack/releases/tag/v0.1.28

Please let me know if you encounter any further issues. Thank you for your report!

**Comment by MrC0D3** [comment]
Thank you for the quick fix! I can confirm the latest version v0.1.28 works as expected.
